.github/actions/fullnode-sync/action.yaml

```diff
@@ -32,12 +32,12 @@ runs:
       run: |
         echo "${HOME}/bin/" >> $GITHUB_PATH # default INSTALL_DIR to path
         scripts/dev_setup.sh -b # Install dependencies
-        sudo apt-get update -y && sudo apt-get install -y expect
+        sudo apt-get update -y && sudo apt-get install
```

Comment:
I think you should remove that line entirely now (the sudo apt stuff)

Reply:
Done. Copy/pasta.